### DISCUSSION TITLE
Docosaurus 3 compatibility

### DISCUSSION THREAD

**Discussion by Vanda53k** [PaloAltoNetworks/docusaurus-openapi-docs#857] [General]
Hi guys.

I'm curious when we can expect plugin to be compatible with docosaurus v3 ?

**Comment by omonk** [comment]
@Vanda53k - it is, i've managed to get it working but you will want to follow the instructions on the README around creating a new project from a git url

https://github.com/PaloAltoNetworks/docusaurus-openapi-docs?tab=readme-ov-file#bootstrapping-from-template-new-docusaurus-site

I started a new project without doing this and wasn't able to get it running but this template seems to work correctly

**Reply by Vanda53k** [reply]
@omonk 

Is it possible to install plugin in already existing project which is not migrated to Typescript? Also, i'm using npm package manager, not yarn.
